pages/linux/netstat.md

```diff
@@ -21,3 +21,7 @@
 - List information continuously:
 
 `netstat -c`
+
+- List routes (-r) without resolve ip to hostname (-n):
+
+`netstat -nr`
```

Comment:
How about matching the order that the options are mentioned in the description? I.e. "-rn".